database/structure.sql

```diff
@@ -0,0 +1,128 @@
+CREATE TABLE users(
+  id SERIAL PRIMARY KEY,
+  username VARCHAR(255),
+  password VARCHAR(255),
+  role_id INTEGER NOT NULL
+);
+
+CREATE TABLE roles(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  create_case BOOLEAN,
+  view_case BOOLEAN,
+  edit_case BOOLEAN,
+  close_case BOOLEAN
+);
+
+CREATE TABLE case_workers(
+  id SERIAL PRIMARY KEY,
+  person_id INTEGER NOT NULL,
+  user_id INTEGER NOT NULL
+);
+
+CREATE TABLE citizens(
+  id SERIAL PRIMARY KEY,
+  person_id INTEGER NOT NULL,
+  cpr INTEGER NOT NULL
+);
+
+CREATE TABLE companies(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  address_id INTEGER NOT NULL
+);
+
+CREATE TABLE addresses(
+  id SERIAL PRIMARY KEY,
+  primary VARCHAR(255),
+  secondary VARCHAR(255),
+  zip_code INTEGER NOT NULL,
+  city VARCHAR(255),
+  country VARCHAR(255)
+);
+
+CREATE TABLE persons(
+  id SERIAL PRIMARY KEY,
+  first_name VARCHAR(255),
+  middle_name VARCHAR(255),
+  last_name INTEGER NOT NULL,
+  phone INTEGER NOT NULL,
```

Comment:
Numbers can contain other characters like `+`, so the VARCHAR type is a better fit here.